hr_holidays_accrual/models/hr_leave_allocation.py

```diff
@@ -0,0 +1,412 @@
+        readonly=True,
+    )
+    limit_accrued_days = fields.Boolean(
+        'Limit Number of Days Accrued',
+        track_visibility='onchange',
+        readonly=True,
+        states={
+            'draft': [('readonly', False)],
+            'confirm': [('readonly', False)]
+        },
+        help=(
+            'Limit total maximum number of days accrued in one accrual'
+            ' period.'
+        ),
+    )
+    max_accrued_days = fields.Float(
+        'Max Number of Days Accrued',
+        track_visibility='onchange',
+        readonly=True,
+        states={
+            'draft': [('readonly', False)],
+            'confirm': [('readonly', False)]
+        },
+        help='Total maximum number of days accrued in one accrual period.',
+    )
+    limit_carryover_days = fields.Boolean(
+        'Limit Number of Days to Carryover',
+        track_visibility='onchange',
+        readonly=True,
+        states={
+            'draft': [('readonly', False)],
+            'confirm': [('readonly', False)]
+        },
+        help=(
+            'Limit total maximum number of days to carryover to next accrual'
+            ' period.'
+        ),
+    )
+    max_carryover_days = fields.Float(
+        'Max Number of Days to Carryover',
+        track_visibility='onchange',
+        readonly=True,
+        states={
+            'draft': [('readonly', False)],
+            'confirm': [('readonly', False)]
+        },
+        help=(
+            'Total maximum number of days to carryover to next accrual period.'
+        ),
+    )
+    limit_accumulated_days = fields.Boolean(
+        'Limit Total Balance',
+        track_visibility='onchange',
+        readonly=True,
+        states={
+            'draft': [('readonly', False)],
+            'confirm': [('readonly', False)]
+        },
+        help='Limit total maximum number of days that can be accumulated.',
+    )
+    max_accumulated_days = fields.Float(
+        'Total Balance Limit',
+        track_visibility='onchange',
+        readonly=True,
+        states={
+            'draft': [('readonly', False)],
+            'confirm': [('readonly', False)]
+        },
+        help='Total maximum number of days that can be accumulated.',
+    )
+    accrual_method = fields.Selection(
+        selection=[
+            ('prorate', 'Prorate'),
+            ('period_start', 'At the beginning of the period'),
+            ('period_end', 'At the end of the period'),
+        ],
+        string='Accrual Method',
+        default='prorate',
+        track_visibility='onchange',
+        readonly=True,
+        states={
+            'draft': [('readonly', False)],
+            'confirm': [('readonly', False)]
+        },
+    )
+    accrual_limit = fields.Integer(
+        compute='_compute_accrual_limit',
+        readonly=True,
+        states={},
+    )
+    number_per_interval = fields.Float(
+        'Allocation per Accrual Period',
+        default=20.0,
```

Comment:
Why 20 ? IMO using an ir.config_parameter or a function to define the default would allow easier override.

Reply:
Good point